scripts/ififuncs.py

```diff
@@ -1265,7 +1265,16 @@ def get_ffmpeg_fmt(path, file_type):
         metadata,
         '-of', 'default=noprint_wrappers=1:nokey=1'
     ]
-    pix_fmt = subprocess.check_output(ffprobe_cmd).rstrip().decode(sys.stdout.encoding).replace("\n", '|').replace("\r", '')
+    try:
+        pix_fmt = subprocess.check_output(ffprobe_cmd).rstrip().decode(sys.stdout.encoding).replace("\n", '|').replace("\r", '')
+    except subprocess.CalledProcessError as grepexc:
+        print(grepexc)
+        man_ffprobe = input("\n !!!\n %s\n Cannot recognize the value to pixel format of this %s file.\n Manually input needed:\n" % (path, file_type))
+        man_ffprobe_yn = input("\n -\n Are you really sure the pixel format of this file is %s?\n enter [y] or [n]\n" % man_ffprobe)
```

Comment:
Pretty sure ififuncs already has “ask yes or no question function”

Reply:
fixed

Reply:
Oh I didn't read the code carefully. Replaced now!